application/comit_node/src/load_settings.rs

```diff
@@ -5,54 +5,40 @@ use structopt::StructOpt;
 
 #[derive(StructOpt, Debug)]
 pub struct Opt {
-    /// Path to configuration folder
+    /// Path to configuration file
     #[structopt(short = "c", long = "config", parse(from_os_str))]
-    config_path: Option<PathBuf>,
+    config_file: Option<PathBuf>,
 }
 
 pub fn load_settings(opt: Opt) -> Result<ComitNodeSettings, ConfigError> {
-    match opt.config_path {
-        Some(config_path) => {
-            let config_path = validate_path(config_path)?;
-            let default_config = Path::join(&config_path, "default");
-            ComitNodeSettings::read(default_config)
+    match opt.config_file {
+        Some(config_file) => {
+            if config_file.exists() {
+                ComitNodeSettings::read(config_file)
+            } else {
+                Err(ConfigError::Message(format!(
+                    "Could not load config file: {:?}",
+                    config_file
+                )))
+            }
         }
         None => match directories::UserDirs::new() {
             None => Err(ConfigError::Message(
                 "Unable to determine user's home directory".to_string(),
             )),
             Some(dirs) => {
-                let config_path = Path::join(dirs.home_dir(), ".config/comit_node/default.toml");
-                if config_path.exists() {
-                    ComitNodeSettings::read(config_path)
+                let config_file = Path::join(dirs.home_dir(), ".config/comit_node/comit_node.toml");
```

Comment:
:mega: The ticket says we want the location to be: `$HOME/.config/comit/comit_node.toml`

Reply:
Updated.